Library/src/Systems/Quota/Quota.cpp

```diff
@@ -231,118 +231,177 @@ void FeaturesQuotaResult::OnResponse(const csp::services::ApiResponseBase* ApiRe
 }
 const csp::common::String TierNameEnumToString(const TierNames& Value)
 {
+	csp::common::String TierString;
```

Comment:
My general belief is that the best log messages are those that help developers debug, diagnose and fix issues. In spirit of that, I would say absolutely.

Fun fact! The web GPU team treat log messages that don't clearly explain the issue and direct developers on how to fix as an actual bug, and track it accordingly.

Reply:
Because now all control paths return a value.

Reply:
The string here is initialised because its default constructor was called. And the CSP string default constructor initialises itself to represent a string of zero length.

If our string class didn't do that in its default constructor though, your concern would be totally valid.

Reply:
This change seems unnecessary. It makes more sense to keep the early returns and just return an empty string for the default case. It should result in cleaner machine code, too.

Reply:
Addressed in latest.